sentry/Cargo.toml

```diff
@@ -11,6 +11,7 @@ Sentry (getsentry.com) client for rust ;)
 """
 edition = "2018"
 autoexamples = true
+rust-version = "1.54.0"
```

Comment:
Wouldn't this have to be 1.60 with this PR?

Reply:
Good catch - was already wondering why the CI didn't throw a human-readable error in the MSRV tests...